controllers/prometheus.go

```diff
@@ -0,0 +1,64 @@
+package controllers
+
+// import (
```

Comment:
any reason this file was all just commented out?

Should it just be deleted if it's no longer used?

Reply:
I added that one in this PR. I plan to use it when adding the new queries. Lint wouldn't let me add those functions since technically they are unused at the moment, so I just commented out the whole file.